### PR TITLE
Done

### PR DESCRIPTION
**Pull request by tracycollins**

### CODE REVIEW

File: src/components/SpiceList.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 // class component
 class SpiceList extends React.Component {
 
-  renderSpices() {
-    // TODO: update this function to return an array of SpiceItem components
-    // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+  renderSpices = (spices) => {
+    return (
+      spices.map(spice => <SpiceItem spice={spice} />)
```
Comment (ihollander) [inline src/components/SpiceList.js:9]: Make sure to include the `key` prop when you are returning an array of JSX, like this:

```jsx
spices.map(spice => <SpiceItem key={spice.id} spice={spice} />)
```

Also, you can access `this.props.spices` directly in your `renderSpices` function, so it's not necessary to pass them in as an argument:


```jsx
renderSpices() {
    return this.props.spices.map(spice => <SpiceItem key={spice.id} spice={spice} />)
}
```

File: src/components/SpiceItem.js [hunk 2]
```diff
@@ -3,11 +3,12 @@ import React from 'react'
...
+    console.log(this.props)
     return (
       <div className="spice-item">
-        <img src={null /* replace me with an image url */} alt={"replace me with a TITLE"} />
+        <img src={this.props.spice.image} alt={this.props.spice.title} />
         <div className="details">
-          <h2>{"replace me with a TITLE"}</h2>
+          <h2>{this.props.title}</h2>
           <p>{"replace me with a DESCRIPTION"}</p>
           <em>{"replace me with NOTES"}</em>
```
Comment (ihollander) [inline src/components/SpiceItem.js:13]: Here, you'd want to use `this.props.spices` to access the spice object and insert the values into your JSX:

```jsx
          <h2>{this.props.spice.title}</h2>
          <p>{this.props.spice.description}</p>
          <em>{this.props.spice.notes}</em>
```

**Comment by ihollander** [comment]
Nice work! I left a couple comments on your code for things that you could work on. Overall though, you've got the right idea 👍

File: src/components/SpiceList.js (hunk 1 above)
Comment (tracycollins) [reply src/components/SpiceList.js:9]: Yes, thanks

File: src/components/SpiceItem.js (hunk 2 above)
Comment (tracycollins) [reply src/components/SpiceItem.js:13]: ok

**Comment by tracycollins** [comment]
Thanks for the review!

**Pull request closed without merging**